client/app/lib/value-format.js

```diff
@@ -1,86 +0,0 @@
-import moment from "moment/moment";
```

Comment:
Was unused?

Reply:
Yes. It was needed only for visualizations. I asked Gabriel why he decided to keep it in the app, and he told me that probably that was accidental. I didn't want to implement same changes second time considering that it will stay unused anyway, and decided to remove this file.